svgscreen/svgscreen.go

```diff
@@ -34,6 +34,17 @@ type BoxSize struct {
 	Height int
 }
 
+type TextSpan struct {
```

Comment:
If these types are internal to the package, should not be instantiatable from outside, you can have the first character lowercase `textSpan` to make it private

Reply:
interesting, is this a style convention thing or does the compiler actually enforce it?

Reply:
It's enforced by the compiler, a bit exotic language design choice :)